src/server.js

```diff
@@ -197,12 +197,11 @@ export class Server {
                     player.isActive = true;
                 } else if (!this.widowUsed){
                     this.widowUsed = true;
+                    player.numLives = 0;
                     this.playersList.getActivePlayer().isActive= false;
                     player.isActive = true;
-                    // Alter player name to make fun of them
-                    player.name = '💩' + player.name.substring(1);
```

Comment:
yesss

Reply:
why you remove??

Reply:
Well, the game is going to be going pretty strongly in the pixel art direction, so the poop emoji is replaced with a pixel art jester's harlequin.

Reply:
let's split the difference: pixel art poop emoji? :D

Reply:
Lots of options :-D

![image](https://user-images.githubusercontent.com/4192460/79492211-85a1f700-7fd4-11ea-8022-4fe76ecc1844.png)

![image](https://user-images.githubusercontent.com/4192460/79492250-8f2b5f00-7fd4-11ea-9434-bcbc35741275.png)

![image](https://user-images.githubusercontent.com/4192460/79492265-97839a00-7fd4-11ea-948c-0ac7b18a8d9b.png)

![image](https://user-images.githubusercontent.com/4192460/79492306-a4a08900-7fd4-11ea-992c-a02552f85b39.png)